src/main/resources/archetype-resources/silo/src/main/java/silo/configuration/VaultCredentialsProvider.java

```diff
@@ -0,0 +1,22 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+package ${package}.${artifactId}.configuration;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.core.env.Environment;
+import org.springframework.vault.annotation.VaultPropertySource;
+
+@Profile("kubernetes")
+@Configuration
+@VaultPropertySource(
+    value = "terraform/\${spring.application.name}",
```

Comment:
@bjoernhaeuser is there a way to use the `mvn archetype:generate` command with a branch? Not sure if I need to escape the `\` here.

Reply:
You can mvn install it then you have a local version which you can use. But I don't think this needs escaping.